plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/utils/RuntimeUtils.java

```diff
@@ -212,7 +212,7 @@ public static String formatExecutionTime(@NotNull Duration duration) {
         } else if (seconds >= 10) {
             return String.format("%ds", seconds);
         } else {
-            return String.format("%d.%ds", seconds, millis / 100);
+            return String.format("%.1fs", seconds + millis / 1000.0);
```

Comment:
Why not just revert what d23a47481edd9f21f314707f841605113b8c681b did?


```suggestion
            return String.format("%d.%03ds", seconds, millis);
```